tensorflow/compiler/xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -1860,12 +1860,14 @@ namespace {
 
 // Returns true if the fusion contains any instruction that is likely
 // translated to complex LLVM IR, such as loops, and prevent vectorization.
-bool MayPreventVectorization(const HloInstruction& hlo) {
+bool MayPreventVectorization(const HloInstruction& hlo,
+                             const bool tolerate_reduce = false) {
```

Comment:
Also could you expand on why do we need this parameter? In general, I'd like to avoid having boolean parameters as much as possible, as they greatly increase the code complexity by doubling the number of possible execution paths.

Reply:
I realized that it isn't needed. So I remove it.